pinot-core/src/test/java/org/apache/pinot/server/realtime/ControllerLeaderLocatorTest.java

```diff
@@ -19,6 +19,7 @@
 package org.apache.pinot.server.realtime;
 
 import org.apache.helix.BaseDataAccessor;
+import org.apache.helix.HelixAdmin;
```

Comment:
Can you please add a test where the resource is disabled, we enable it, and then disable it, and make sure that the right code path s invoked in these cases?

Reply:
We have such tests in `PinotControllerModeTest`